frontend/lib/api.ts

```diff
@@ -1,4 +1,4 @@
-import { FormData } from './components/Form/SignupForm'
+import { FormData } from '../src/components/Form/SurveyForm'
```

Comment:
normally you should to expose form data as external interface outside SurveyForm

Reply:
but in this case this is temporary